src/components/game-board/board.tsx

```diff
@@ -0,0 +1,17 @@
+import { component$, useStylesScoped$ } from "@builder.io/qwik";
+import { BoardColumn } from "../board-column/column";
+import styles from "./board.css?inline";
+
+export const GameBoard = component$(() => {
+	useStylesScoped$(styles);
+
+	const renderColumns = () => {
+		const columns = [];
+		for (let i = 0; i < 10; i++) {
```

Comment:
[**A comment** (PR comments with a purpose)](https://gist.github.com/dgoings/ecfe4877b5992190e4fc8893c77a7ae6):

We got a magic number here! `10` This should also get fixed with addition of state.